examples_tests/60.ClusteredRendering/lighting.frag

```diff
@@ -0,0 +1,270 @@
+#version 430 core
```

Comment:
factor out the fragment shader agnostic stuff and put in `nbl/builtin/glsl/ext/clusteredlighting/lighting.glsl`add appropriate #ifdef/#define guards and `nbl_glsl_ext_ClusteredLighting_` prefixes to declarations, preprocessor directives and variable names

Reply:
some forward declared `get`ters might be necessary